src/bindings/js/node/tests/unit/basic.test.js

```diff
@@ -186,60 +195,53 @@ describe('ov basic tests.', () => {
         assert.strictEqual(obj.output().getAnyName(), 'fc_out');
         assert.strictEqual(obj.output().anyName, 'fc_out');
       });
-    });
 
+    });
   });
 
-  describe('Input class for ov::Input<const ov::Node>', () => {
+  describe('ov.Model/ov.CompiledModel input()', () => {
 
-    it('Output type', () => {
+    it('ov.Model.input() type', () => {
       assert.ok(model.input() instanceof ov.Output);
     });
 
-    it('ConstOutput type', () => {
+    it('ov.CompiledModel.input() type', () => {
       assert.ok(compiledModel.input() instanceof ov.ConstOutput);
     });
 
-    modelLike.forEach(([obj]) => {
-      it('input() is typeof object', () => {
-        assert.strictEqual(typeof obj.input(), 'object');
-      });
+    it('input() methods/properties', () => {
+      modelLike.forEach((obj) => {
 
-      it('inputs property', () => {
+        assert.strictEqual(typeof obj.input(), 'object');
         assert.strictEqual(obj.inputs.length, 1);
-      });
-
-      it('input().toString()', () => {
         assert.strictEqual(obj.input().toString(), 'data');
-      });
-
-      it('input(idx: number).ToString() method', () => {
         assert.strictEqual(obj.input(0).toString(), 'data');
-      });
-
-      it('input(tensorName: string).ToString() method', () => {
         assert.strictEqual(obj.input('data').toString(), 'data');
-      });
-
-      it('input().getAnyName() and anyName', () => {
         assert.strictEqual(obj.input().getAnyName(), 'data');
         assert.strictEqual(obj.input().anyName, 'data');
-      });
 
-      it('input(idx).shape property with dimensions', () => {
         assert.deepStrictEqual(obj.input(0).shape, [1, 3, 32, 32]);
         assert.deepStrictEqual(obj.input(0).getShape(), [1, 3, 32, 32]);
       });
+
     });
 
   });
 
   describe('Test exportModel()/importModel()', () => {
-    const userStream = compiledModel.exportModelSync();
     const epsilon = 0.5;
     const tensor = Float32Array.from({ length: 3072 }, () => (Math.random() + epsilon));
```

Comment:
It can be also moved into `before` block. (probably it makes sense define `epsilon` at the top of the module and `tensor` inside before, it's up to you)